src/modules/cmdpal/Exts/Microsoft.CmdPal.Ext.WindowWalker/Commands/SwitchToWindowCommand.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CmdPal.Ext.WindowWalker.Components;
+using Microsoft.CmdPal.Ext.WindowWalker.Properties;
+using Microsoft.CmdPal.Extensions;
+using Microsoft.CmdPal.Extensions.Helpers;
+
+namespace Microsoft.CmdPal.Ext.WindowWalker.Commands;
+
+internal sealed partial class SwitchToWindowCommand : InvokableCommand
```

Comment:
IIRC, we have this in the Process Monitor extension too, but this seems like it's a strictly better replacement for that. 

Maybe we should follow up by deleting that one?

Reply:
I think the process monitor one needs a lot of TLC before it becomes a real thing we want people to use. 

This is def better flow. Not going to address this in this PR naturally.